lib/README.md

```diff
@@ -1,10 +0,0 @@
-EasyRPG Player
-==============
-
-The following note only applies to these builds:
-
-- Visual Studio 2015 project (Windows)
-- OpenDingux Makefile (Dingoo, GCW Zero)
```

Comment:
This will break the current GCW0 build job. However, should be ok to put liblcf as shared library in the OPK or just link it in as static library.

Reply:
oh, I assumed that this OpenDingux note is outdated...